### DISCUSSION TITLE
How to deploy contract/bytecode using `forge` or `cast`?

### DISCUSSION THREAD

**Discussion by sebastiantf** [foundry-rs/foundry#1224] [Q&A] [answered]
How can I deploy a contract with just the bytecode using `forge` or `cast`?

I'm looking for an alternative for `seth send --create`

**Comment by onbjerg** [comment, marked as answer]
If you just want to deploy raw bytecode, there is no nice way to do this at the moment, unfortunately.

It is one of the remaining features in order for us to reach `seth` parity (see #29)

**Reply by sebastiantf** [reply]
Got it. Thanks!

**Comment by nicolasembleton** [comment]
It seems you can now, though I haven't gotten it to work yet and the documentation is a bit sparse in examples.

See: https://github.com/foundry-rs/foundry/pull/742

**Reply by 0ShadowKitty0** [reply]
Try with: 

`cast send  --from <local_anvil_address> --value 10ether --unlocked --create 0000000000000000000000000000000000000000000000000000000000000001 "<contract_creation_bytecode>"`

in this sample the contract constructor needs a uint256 which I set as 1
